taskcluster/fenix_taskgraph/transforms/github_release.py

```diff
@@ -0,0 +1,74 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+"""
+Apply some defaults and minor modifications to the jobs defined in the github_release
+kind.
+"""
+
+from __future__ import absolute_import, print_function, unicode_literals
+
+import os
+import codecs
+
+from taskgraph.transforms.base import TransformSequence
+from taskgraph.util.schema import resolve_keyed_by
+
+
+transforms = TransformSequence()
+
+
+@transforms.add
+def resolve_keys(config, tasks):
+    for task in tasks:
+        for key in ("worker.is-prerelease", "worker.dep"):
+            resolve_keyed_by(
+                task,
+                key,
+                item_name=task["name"],
+                **{
+                    'build-type': task["attributes"]["build-type"],
+                }
+            )
+        yield task
+
+
+@transforms.add
+def build_worker_definition(config, tasks):
+    for task in tasks:
+        worker_definition = {
+            "artifact-map": _build_artifact_map(task),
+            "git-tag": config.params["head_tag"].decode("utf-8"),
+            "git-revision": config.params["head_rev"].decode("utf-8"),
+            "release-name": task["worker"]["release-name"].format(version=config.params["version"]),
+            "release-description": _get_release_notes(config),
+        }
+
+        task["worker"].update(worker_definition)
+
+        yield task
+
+def _build_artifact_map(task):
```

Comment:
+1 for the consistency with beetmover schemas, even though they are not related in practice, but only conceptually.